Gems/Atom/Asset/ImageProcessingAtom/Code/Source/BuilderSettings/BuilderSettingManager.cpp

```diff
@@ -599,7 +607,14 @@ namespace ImageProcessingAtom
         size_t lastUnderScore = fileName.find_last_of(FileMaskDelimiter);
         if (lastUnderScore != AZStd::string::npos)
         {
-            return fileName.substr(lastUnderScore);
+            // If the complete file name contains multiple extension separators ('.'), the GetFileName() call above will
```

Comment:
We may use QFileInfo which we can just use baseName() to get the file name without any '.' parts at line 604.

Reply:
Great suggestion, I'll change this and also simplify the explanatory comment. :)